### PR TITLE
Changes to abstract out InstumentationKey from Properties.json to a separate file

### PR DESCRIPTION
**Pull request by suskumar-MSFT**
Changes to abstract out InstumentationKey from Porperties.json to a separate file - ApplicationInsightsKey.json

### CODE REVIEW

File: src/PAModel/Schemas/adhoc/AppInsightsKey.cs [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+namespace Microsoft.PowerPlatform.Formulas.Tools
+{
+    // Abstract out the InstrumentationKey (From AppInsights) of an app in a seprate json file (AppInsightsKey.json)
+    class AppInsightsKeyJson
+    {
+        public string InstrumentationKey { get; set; } // a guid
+
+        [JsonExtensionData]
+        public Dictionary<string, JsonElement> ExtensionData { get; set; }
```
Comment (MikeStall) [inline src/PAModel/Schemas/adhoc/AppInsightsKey.cs:18]: I think we can skip this - this json file is only in the extract case, so we control all the fields here.

Comment (MikeStall) [reply src/PAModel/Schemas/adhoc/AppInsightsKey.cs:18]: put another way, any extra fields here would *not* be added into the msapp.

Comment (suskumar-MSFT) [reply src/PAModel/Schemas/adhoc/AppInsightsKey.cs:18]: yea I was thinking the same, will remove it.

File: src/PAModel/Serializers/MsAppSerializer.cs [hunk 2]
```diff
@@ -274,6 +274,11 @@ public static CanvasDocument Load(Stream streamToMsapp, ErrorContainer errors)
                     app._properties.LocalDatabaseReferences = null;
                     app._entropy.LocalDatabaseReferencesAsEmpty = false;
                 }
+                if (!string.IsNullOrEmpty(app._properties.InstrumentationKey))
```
Comment (MikeStall) [inline src/PAModel/Serializers/MsAppSerializer.cs:277]: When we roundtrip, we'll need to preserve between null and "".

**Review by MikeStall — APPROVED** [review]

**Pull request merged**